src/violations.rs

```diff
@@ -870,6 +870,21 @@ impl Violation for GlobalVariableNotAssigned {
     }
 }
 
+define_violation!(
+    pub struct TooManyArgs {
+        pub c_args: usize,
+        pub max_args: usize,
+    }
+);
+
+impl Violation for TooManyArgs {
+    #[derive_message_formats]
+    fn message(&self) -> String {
+        let TooManyArgs { c_args, max_args } = self;
+        format!("Too many arguments ({c_args}/{max_args})")
+    }
+}
```

Comment:
Can you move this into `src/rules/pylint/rules/too_many_args.rs`? We're trying to colocate violations with their rule implementations going forward.